core/src/main/scala/org/ensime/util/FileUtil.scala

```diff
@@ -33,7 +33,11 @@ object RichFileObject {
 object FileUtils {
 
   def isScalaSourceFile(f: File): Boolean = {
-    f.exists && f.getName.endsWith(".scala")
+    f.getName.endsWith(".scala")
```

Comment:
Cool, fair enough

> On 7 Oct 2015, at 00:41, Aemon Cannon <notifications@github.com> wrote:
> 
> In core/src/main/scala/org/ensime/util/FileUtil.scala:
> 
> > @@ -33,7 +33,11 @@ object RichFileObject {
> >  object FileUtils {
> >  
> >    def isScalaSourceFile(f: File): Boolean = {
> > -    f.exists && f.getName.endsWith(".scala")
> > +    f.getName.endsWith(".scala")
> IMO it's surprising that this would hit the disk. Probably a (my) mistake in the first place. Only one client of this method and it looks safe to change.
> 
> —
> Reply to this email directly or view it on GitHub.
>